Makefile

```diff
@@ -34,7 +34,7 @@ GOLANG_VERSION = 1.11
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*" -not -path "./client/*")
 
 .PHONY: up
-up: vendor config/dex.yml start config/config.toml ## Set up the development environment
+up: vendor config/dex.yml start config/config.toml ## Set up the development environment (override existing configuration)
```

Comment:
It does not override existing configuration, creates configuration if there is none.

Reply:
Right, no source files for configs targets. I always want to run `make up` instead of `make start`, but only remembered it's not what I should do, but the reason is not that it overrides config files but that it runs an unnecessary vendoring.